cfme/tests/test_rest.py

```diff
@@ -371,6 +702,43 @@ def test_vm_add_lifecycle_event(request, provider, rest_api, vm, from_detail, db
     ))) == 1, "Could not find the lifecycle event in the database"
 
 
+@pytest.mark.parametrize(
+    "multiple", [True, False],
+    ids=["one_request", "multiple_requests"])
```

Comment:
Aren't the ids swapped? First one should be multiple and the other not, is that correct?